contributors/devel/api-conventions.md

```diff
@@ -123,9 +123,12 @@ filtering (see [the labels documentation](https://kubernetes.io/docs/user-guide/
 lists should support filtering by fields.
 
    Examples: PodLists, ServiceLists, NodeLists
-
-   TODO: Describe field filtering below or in a separate doc.
-
+   
+   *Equality-based filtering by label keys and values.*
+   ``` NodeLists != DaemonSet-foo ```  
```

Comment:
this doesn't really make sense... I think the TODO was looking for an example API call that demonstrated field filtering

Reply:
+1 I don't understand the new text and it is not fulfilling the TODO

Reply:
My understanding for the TODO - provide use case of field filtering.  Under that assumption  Daemonset-foo would not be in  NodeLists. ie a Daemonset would be set to to more specific set of Nodes like Nodelists-bar.  It may make more sense to include an API call instead.